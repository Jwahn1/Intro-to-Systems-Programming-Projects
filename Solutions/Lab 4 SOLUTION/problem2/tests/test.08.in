new 10
add hello
add world
add byebye
remove 1
remove 0
add adieu
remove 1
end
