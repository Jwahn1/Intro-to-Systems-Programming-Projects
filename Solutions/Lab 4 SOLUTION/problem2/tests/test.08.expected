(0/10) [ ]
(1/10) [ hello ]
(2/10) [ hello world ]
(3/10) [ hello world byebye ]
(2/10) [ hello byebye ]
(1/10) [ byebye ]
(2/10) [ byebye adieu ]
(1/10) [ byebye ]
